src/Kunstmaan/MediaBundle/DependencyInjection/KunstmaanMediaExtension.php

```diff
@@ -16,6 +16,16 @@
  */
 class KunstmaanMediaExtension extends Extension implements PrependExtensionInterface
 {
+    const DEFAULT_CROPPING_CONFIG = [
+        'default' => [
+            ['name' => 'desktop', 'width' => 1, 'height' => 1, 'lockRation' => true],
```

Comment:
Typo
```suggestion
            ['name' => 'desktop', 'width' => 1, 'height' => 1, 'lockRatio' => true],
```

And also rename this to `lock_ratio` when the configuration is changed